lcidlc/src/Interface.cpp

```diff
@@ -180,8 +180,20 @@ static bool InterfaceReport(InterfaceRef self, Position p_where, InterfaceError
 	case kInterfaceErrorJavaImpliesNonIndirectReturn:
 		fprintf(stderr, "Java mapped methods cannot have indirect return value\n");
 		break;
+	case kInterfaceErrorMethodsCannotHaveVariants:
+		fprintf(stderr, "Java mapped methods cannot have variants\n");
+		break;
+	case kInterfaceErrorMethodsAreAlwaysTail:
+		fprintf(stderr, "Java mapped methods are always tail\n");
+		break;
+	case kInterfaceErrorMethodsMustBeJava:
+		fprintf(stderr, "Java mapped methods must be java\n");
+		break;
+	case kInterfaceErrorNone:
+		MCUnreachableReturn(false);
```

Comment:
Surely this should be something like:

```
fprintf(stderr, "No error occurred\n");
```

Aborting because there wasn't an error seems a little bit extreme.